src/com/owncloud/android/utils/DisplayUtils.java

```diff
@@ -207,8 +207,17 @@ else if ((System.currentTimeMillis() - time) < 60 * 1000) {
                 dateString = DateUtils.getRelativeDateTimeString(c, time, minResolution, transitionResolution, flags);
             }
         }
-        
-        return dateString.toString().split(",")[0];
+
+        String[] parts = dateString.toString().split(",");
+        if (parts.length == 2) {
+            if (parts[1].contains(":") && !parts[0].contains(":")) {
+                return parts[0];
+            } else if (parts[0].contains(":") && !parts[1].contains(":")) {
+                return parts[1];
+            }
+        }
+        //dateString contains unexpected format. use localized, absolute date.
+        return DisplayUtils.unixTimeToHumanReadable(time);

```

Comment:
But looking at @rperezb screenshot for Android 6 you end up with a weird result like "15.09" ?!